src/dbnode/encoding/iterator_test.go

```diff
@@ -86,6 +87,8 @@ func (it *testIterator) Reset(r io.Reader) {
 	it.onReset(r)
 }
 
+func (it *testIterator) SetSchema(descr namespace.SchemaDescr) {}
```

Comment:
If schema was passed in Reset then you can just ignore the arg, rather than needing to add no-op methods to these types.

Reply:
fair